pkg/api/v1/router_server_test.go

```diff
@@ -522,6 +523,7 @@ func TestIntegrationServerGetDeleted(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, r.UUID, uuid.MustParse(dbtools.FixtureChuckles.ID), "Expected UUID %s, got %s", dbtools.FixtureChuckles.ID, r.UUID.String())
 			assert.Equal(t, r.Name, dbtools.FixtureChuckles.Name.String)
+			assert.NotEqual(t, r.DeletedAt, null.Time{}.Time)
```

Comment:
Mostly curious, does `assert.NotNil` not work here?

Reply:
I'm specifically looking to see that it sets it to the zero value defined by the null package, but I guess its leverage language mechanics, since `null.Time.Time` is a `time.Time` type. 

🤷

Reply:
I tried Nil, since that's really what I'm looking for, and that results in test failure when I would expect it to pass.

Reply:
yeah....thinking about it this morning a `null.Time` isn't going to be nil....so makes sense that it would fail